### ISSUE TITLE
Accessibility: exit button uses prohibited ARIA attribute

### ISSUE THREAD

**Issue by rkarwacki** [ain/smartbanner.js#818]
__Steps to reproduce__:

1. Configure Smartbanner for a live webpage and provide the [aria-label text](https://github.com/ain/smartbanner.js/blob/81e6e7ce43b9f0f412e82ea6e7d53b82b990d033/src/smartbanner.js#L156) for the button that closes the banner
2. Run the [Google PageSpeed report](https://pagespeed.web.dev/) for this webpage
3. Switch to the mobile report 
4. Scroll down to the accessibility section

__What happens__:
An accessibility issue is detected for the <span> element that defines the close button for smartbanner:
<img width="980" alt="Screenshot 2024-10-22 at 17 52 15" src="https://github.com/user-attachments/assets/d59327b2-614e-42b8-97f4-803b720c85e1">
"Elements use prohibited ARIA attributes. Using ARIA attributes in roles where they are prohibited can mean that important information is not communicated to users of assistive technologies. [Learn more about prohibited ARIA roles](https://dequeuniversity.com/rules/axe/4.10/aria-prohibited-attr)."

__What should happen__:
No accessibility issues are detected during the scan. 

The `<span>` element should have a proper aria role. Alternatively, the `<span>` should be replaced with a `<button>` element which has semantic meaning, unlike the `<span>`. Note that there is a `button` aria-role ([reference](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role)), but the docs suggest using a <button> or <input> element instead.

**Comment by ain** [comment]
Issue confirmed, thank you @rkarwacki! 

Goes without the saying, that PR is welcome!

**Comment by ain** [comment]
For reference, the issue was introduced by #721

**Comment by rkarwacki** [comment]
@ain as the issue is confirmed, could you please leave the issue open? I will try to provide a fix.

**Comment by ain** [comment]
Issue was closed as per the fix and the eventual release yesterday. 

Unless it's not working, I'd keep the issue closed.

**Comment by rkarwacki** [comment]
@ain oh, I'm sorry, I didn't realise the issue was fixed. Thank you!
